src/components/labelled-date-input/LabelledDateInput.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import DateTime from 'react-datetime'
+import 'react-datetime/css/react-datetime.css'
+import moment from 'moment'
+import 'moment/locale/ru'
+import './LabelledDateInput.css'
+
+const LabelledDateInput = () => {
```

Comment:
Lets implement this component this way:
```javascript
 const LabelledDateInput = (props) => (
     <LabelledInput
        labelClassName="labelled-date-input__label"
        renderInput={() => (<DateTime timeFormat={false} defaultValue={moment()} locale="ru" />)}
        {...props}
      />)
```
Cause currently `LabelledDateInput` has no label, what is misleading. And you need to use same code in every place, where you want to use this component with label